src/dvc_objects/fs/base.py

```diff
@@ -573,21 +573,35 @@ def info(
         path: AnyFSPath,
         callback: fsspec.Callback = ...,
         batch_size: Optional[int] = ...,
+        return_exceptions: bool = ...,
         **kwargs,
-    ) -> "Entry": ...
+    ) -> Union["Entry", Exception]: ...
```

Comment:
@falko17, it looks like this change requires a few more specialized overloads.

Could you please handle that? Thanks. 🙂